### PR TITLE
update to latest xplot.plotly library

### PR DESCRIPTION
**Pull request by colombod**

### CODE REVIEW

File: XPlot.DotNet.Interactive.KernelExtensions/XPlot.DotNet.Interactive.KernelExtensions.csproj [hunk 1]
```diff
@@ -8,7 +8,7 @@
   <ItemGroup>
     <PackageReference Include="FSharp.Core" />
```
Comment (cartermp) [inline XPlot.DotNet.Interactive.KernelExtensions/XPlot.DotNet.Interactive.KernelExtensions.csproj]: should be `Update` across the board I think

**Comment by cartermp** [comment]
SHortly https://www.nuget.org/packages/XPlot.Plotly/ should point to 3.0.1 which has `>=` as an FSharp.Core dependency, which should resolve this

**Comment by colombod** [comment]
> SHortly https://www.nuget.org/packages/XPlot.Plotly/ should point to 3.0.1 which has `>=` as an FSharp.Core dependency, which should resolve this

Done INTOTHENIGHT

**Comment by cartermp** [comment]
we're gonna totes m'goats yeet this dependency update

**Pull request merged**